sopel_modules/rat_board.py

```diff
@@ -1648,9 +1648,19 @@ def cmd_reopen(bot, trigger, id):
     """
     try:
         result = callapi(bot, 'PUT', data={'status': 'open'}, uri='/rescues/' + str(id), triggernick=str(trigger.nick))
-        refresh_cases(bot, force=True)
-        updateBoardIndexes(bot)
-        bot.say('Reopened case. Cases refreshed, care for your case numbers!')
+        try:
+            addNamesFromV2Response(result['included'])
+        except:
+            pass
```

Comment:
Yep I agree that this is terribly bad style of code. This is just how this is done in the rest of the code and I couldn't <span style="color:grey"> (be bothered to) </span> figure out what addNamesFromV2Response actually throws.

Reply:
From what i can tell it shouldn't be raising an exception. and if it does methinks we would like to know about it. Catch/pass is awful code style. I would prefer if mecha raised an error rather than silence it, that's pretty much the only indication we get if something is broken.

There are a lot of things you may discover in mecha2's codebase that are just plain bad practice. 
For now it would be appreciated that any method you introduce/modify you make attempt to clean up. 

We will just have to maintain mecha2 until mecha3 is ready to go. (which may be a while)

Reply:
Yeah you're right, I'll have a look at fixing that except clause tomorrow.

Reply:
Also, is Mecha3 already on its way to becoming a thing? As in a complete rewrite against `pydle` or similar?